### PR TITLE
Updated lib for androidx

### PR DESCRIPTION
**Pull request by edferreira**
I had to made some changes to your lib for use in my project, so I'm creating this PR hoping it can help you to maintain the lib

The changes consist into updating some .java to new androidx and other minor fixes

I also had to add some functions to it to retrieve a list of images. The function receives a source image, start time, end time and a step and returns a list of images of those timestamps

### CODE REVIEW

File: android/src/main/java/com/shahenlibrary/Trimmer/Trimmer.java [hunk 1]
```diff
@@ -387,11 +387,13 @@ static void trim(ReadableMap options, final Promise promise, ReactApplicationCon
...
     cmd.add("ultrafast");
     // NOTE: DO NOT CONVERT AUDIO TO SAVE TIME
+    cmd.add("-c:v");
+    cmd.add("copy");
     cmd.add("-c:a");
     cmd.add("copy");
     // NOTE: FLAG TO CONVER "AAC" AUDIO CODEC
-    cmd.add("-strict");
-    cmd.add("-2");
+    // cmd.add("-strict");
```
Comment (shahen94) [inline android/src/main/java/com/shahenlibrary/Trimmer/Trimmer.java]: Why these lines are commented?

File: android/src/main/java/com/shahenlibrary/Trimmer/Trimmer.java [hunk 2]
```diff
@@ -603,6 +605,64 @@ static void getPreviewImageAtPosition(String source, double sec, String format,
     promise.resolve(event);
   }
 
+  static void getTrimmerPreviewImages(String source, double startTime, double endTime, int step, String format, final Promise promise, ReactApplicationContext ctx) {
```
Comment (shahen94) [inline android/src/main/java/com/shahenlibrary/Trimmer/Trimmer.java]: Actually this method is not related to the androidx PR, anyway, we'll merge it

Comment (edferreira) [reply android/src/main/java/com/shahenlibrary/Trimmer/Trimmer.java]: not, those aren't related. Just bundled everything together for the PR, sorry

File: android/src/main/java/com/shahenlibrary/Trimmer/Trimmer.java (hunk 1 above)
Comment (edferreira) [reply android/src/main/java/com/shahenlibrary/Trimmer/Trimmer.java]: -strict -2 doesn't seem needed any more.

I added the video copy for performance, was having trouble with android and after adding the -c:v copy the runtime had a huge improvement
